Update path used in asset filter example

bschulzebaek opened the pull request:
Updated the asset path used in the code example, this has changed during 6.7 / vite changes.

Also removed the warning about Vue filters, as our `assetFilter`, which is described right above the warning is not an actual Vue filter and thus not affected by the deprecation - quite confusing. Devs should be able to learn about Vue 2 filter deprecation in their respective docs, which are linked or referred to throughout our upgrade guides anyway 💡

Isengo1989 commented:
@bschulzebaek Github did mix up smth I suppose. I change the target branch to next-6.7 and after rebase the PR is now crooked (adr changes that are not supposed to be there). If you want to recreate the PR with target "next-6.7" please do so, if you don't have time, I can take care of it.

bschulzebaek commented:
> @bschulzebaek Github did mix up smth I suppose. I changed the target branch to next-6.7, and after rebase, the PR is now crooked (adr changes that are not supposed to be there). If you want to recreate the PR with target "next-6.7" please do so, if you don't have time, I can take care of it.

No worries, fixed it with a `git reset` 😄

bschulzebaek commented:
@Isengo1989 am I missing something for this PR, or can it be merged? Guess I have no permission to merge into next-6.7 in this repo 🤔

Isengo1989 commented:
@bschulzebaek nope, just the approval that came on my vacation day on Monday :slightly_smiling_face: 

Merged :+1:
